ocs_ci/ocs/node.py

```diff
@@ -1626,8 +1626,8 @@ def verify_all_nodes_created():
             )
         elif (
             config.ENV_DATA["platform"].lower() == constants.VSPHERE_PLATFORM
-            and config.ENV_DATA["deployment_type"] == "ipi"
-        ):
+            or config.ENV_DATA["platform"].lower() == constants.IBMCLOUD_PLATFORM
```

Comment:
I think we can add also Azure as I saw lately failing with also 5 nodes out of 6.
And do something like: config.ENV_DATA["platform"].lower() in [list_of_platform]

Reply:
done